package.json

```diff
@@ -37,58 +70,30 @@
     "stylelint-config-standard": "^17.0.0",
     "supertest": "^3.0.0",
     "testpilot-ga": "^0.3.0",
-    "uglifyify": "^4.0.3"
+    "uglifyify": "^4.0.3",
```

Comment:
These are in dev dependencies because they are browserified or copied to /public in as a build step. Only the dependencies required by the server (express) are not devDependencies in order to make the docker image used in production as small as possible.